cincinnati/Cargo.toml

```diff
@@ -31,16 +31,16 @@ async-trait = "^0.1"
 tempfile = "^3.8.0"
 flate2 = "^1.0.34"
 tar = "^0.4.40"
-dkregistry = { git = "https://github.com/camallo/dkregistry-rs.git", rev = "0a3c99b7ede0c177a4389e0ffd9e566572633f8c" }
+dkregistry = { git = "https://github.com/hongkailiu/dkregistry-rs.git", rev = "11b974e1703987f837dfee1b68e73316c0348dd2" }
```

Comment:
For testing only

/hold

Wait for https://github.com/camallo/dkregistry-rs/pull/265

Reply:
Done.